TimberAPI/AssetLoaderSystem/ResourceAssetPatch/TimberApiResourceAssetLoader.cs

```diff
@@ -11,15 +10,30 @@ public class TimberApiResourceAssetLoader : IResourceAssetLoader
     {
         private readonly IAssetLoader _assetLoader;
 
-        private Shader _shader;
+        private static Shader _buidlingShader;
+        private static Shader _dirtShader;
+        private static Shader _pathShader;
 
-        private Shader Shader
+        private static Shader Shader(Material mat)
         {
-            get
+           
+            if (_buidlingShader == null)
+                _buidlingShader = Resources.Load<GameObject>("Buildings/Paths/Platform/Platform.Full.Folktails").GetComponent<MeshRenderer>().materials[0].shader;
+            if (_dirtShader == null)
+                _dirtShader = Resources.Load<GameObject>("Buildings/Wellbeing/Carousel/CarouselConstructionSite5x5+3x1Dirt").GetComponentsInChildren<MeshRenderer>()[1].materials[0].shader;
```

Comment:
Just curious, why are we getting the second MesRenderer in `.GetComponentsInChildren<MeshRenderer>()[1]` and not the first?

Reply:
That is because there is no prefab that has dirt as its main material. That is why I chose this prefab as its the closest to an mesh render that has the dirt material.